l10n/gitahead_ru.ts

```diff
@@ -0,0 +1,4999 @@
+    </message>
+    <message>
+        <location filename="../src/update/UpdateDialog.cpp" line="75"/>
+        <source>Remind Me Later</source>
+        <translation>Напомнить мне позже</translation>
+    </message>
+    <message>
+        <location filename="../src/update/UpdateDialog.cpp" line="77"/>
+        <source>Skip This Version</source>
+        <translation>Пропустить эту версию</translation>
+    </message>
+</context>
+<context>
+    <name>UpdatePanel</name>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="619"/>
+        <source>Check for updates automatically</source>
+        <translation>Проверять обновления автоматически</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="626"/>
+        <source>Automatically download and install updates</source>
+        <translation>Автоматически загружать и ставить обновления</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="633"/>
+        <source>Check Now</source>
+        <translation>Проверить сейчас</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/SettingsDialog.cpp" line="638"/>
+        <source>Software Update:</source>
+        <translation>Обновление ПО:</translation>
+    </message>
+</context>
+<context>
+    <name>UpdateSubmodulesDialog</name>
+    <message>
+        <location filename="../src/dialogs/UpdateSubmodulesDialog.cpp" line="122"/>
+        <source>Recursive</source>
+        <translation>Рекурсивно</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/UpdateSubmodulesDialog.cpp" line="125"/>
+        <source>Init</source>
+        <translation>Инициализировать</translation>
+    </message>
+    <message>
+        <location filename="../src/dialogs/UpdateSubmodulesDialog.cpp" line="130"/>
+        <source>Update</source>
+        <translation>Обновить</translation>
+    </message>
+</context>
+<context>
+    <name>Updater</name>
+    <message>
+        <location filename="../src/update/Updater_win.cpp" line="24"/>
+        <source>Installer failed to start</source>
+        <translation>Установщик не смог запуститься</translation>
+    </message>
+    <message>
+        <location filename="../src/update/Updater_mac.mm" line="77"/>
+        <source>The disk image failed to mount successfully</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/update/Updater_mac.mm" line="96"/>
+        <source>The existing bundle could not be moved to the trash</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/update/Updater_mac.mm" line="108"/>
+        <source>The new bundle could not be copied into place</source>
+        <translation type="unfinished"></translation>
```

Comment:
?

Reply:
The translation is unfinished. I have translated only short texts.

Reply:
Fixed.